docker/parabol-ubi/docker-build/environments/pipeline

```diff
@@ -47,11 +48,11 @@ RETHINKDB_URL='rethinkdb://localhost:28015/actionProduction'
 SEGMENT_WRITE_KEY=''
 SEGMENT_FN_KEY=''
 SENTRY_DSN=''
-SERVER_ID=''
+SERVER_ID='0'
 SERVER_SECRET='FAKE_VALUE'
 SLACK_CLIENT_ID=''
 SLACK_CLIENT_SECRET=''
-STRIPE_PUBLISHABLE_KEY=''
+STRIPE_PUBLISHABLE_KEY='pk_test_MNoKbCzQX0lhktuxxI7M14wd'
```

Comment:
keys added here are required for running tests. this stripe key is NOT a secret & it goes to our stripe testing env